spec/models/user_spec.rb

```diff
@@ -1,5 +1,48 @@
 require 'rails_helper'
-
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
+  
+  # Creando instancias de User y de Tweet usando FactoryBot
+  let(:user) { create(:user) }
+  let(:tweet) { create(:tweet, user: user) }
+  let(:another_user) { create(:user) }
+
+  describe 'validations' do
+    it { should validate_presence_of(:username) }
+    it { should validate_uniqueness_of(:username).case_insensitive }
+    it { should validate_presence_of(:email) }
+    it { should validate_uniqueness_of(:email).case_insensitive }
+    it { should validate_presence_of(:password) }
+    it { should allow_value('valid_Password123').for(:password)}
+    it { should validate_length_of(:password).is_at_least(12)}
+  end
+  
+  describe 'associations' do
+    it { should have_many(:tweets) }
+    it { should have_many(:retweets) }
+    it { should have_many(:followers) }
+    it { should have_many(:followees) }
```

Comment:
You missed to check the has_many through relationships for `followees`, `followers` and `liked_tweets`